content/applications/finance/fiscal_localizations/peru.rst

```diff
@@ -2,33 +2,77 @@
 Peru
 ====
 
-.. |SUNAT| replace:: :abbr:`SUNAT (Superintendencia Nacional de Aduanas y de Administración Tributaria)`
+.. |SUNAT| replace:: :abbr:`SUNAT (Superintendencia Nacional de Aduanas y de Administración
+   Tributaria)`
 .. |GRE| replace:: :abbr:`GRE (Guía de Remisión Electrónica)`
 .. |RUS| replace:: :abbr:`RUS (Régimen Único Simplificado)`
 .. |EDI| replace:: :abbr:`EDI (Electronic Data Interchange)`
 
-Introduction
-============
+Modules
+=======
+
+:ref:`Install <general/install>` the following modules to utilize all the current features of the
+Peruvian localization.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Technical name
+     - Description
+   * - :guilabel:`Chile - Accounting`
+     - `l10n_pe`
+     - Adds accounting features for the Peruvian localization, which represent the minimal
+       configuration required for a company to operate in Peru and under the SUNAT regulations and
+       guidelines. The main elements included in this module are: chart of account, taxes, document
+       types.
+   * - :guilabel:`Peru - E-invoicing`
+     - `l10n_pe_edi`
+     - Includes all technical and functional requirements to generate and receive electronic
+       invoices online based on the SUNAT regulations.
+   * - :guilabel:`Peru - Accounting Reports`
+     - `l10n_pe_reports`
+     - Includes the following financial reports.
+
+       - 1st set of main financial reports:
+
+         - Registro de Ventas e Ingresos (RVIE) - 14.4
+         - Registro de Compras Electrónico (RCE) - 8.4
+         - Registro de Compras Electrónico - Información de Operaciones con Sujetos no Domiciliados
+           (RCE) - 8.5
+
+       - 2nd set of financial reports:
+
+         - PLE 5.1 General Journal
+         - PLE 5.3 Chart of Accounts
+         - PLE 6.1 General Ledger
+
+       - 3rd set of financial reports:
+
+         - PLE 1.1 Libro de Caja
+         - PLE 1.2 Libro de Bancos
+
+   * - :guilabel:`Peruvian - Electronic Delivery Note`
+     - `l10n_pe_edi_stock`
+     - The delivery guide (Guía de Remisión) is needed as proof that you are sending goods between A
+       and B. It is only when a delivery order is validated that you can create the delivery guide.
```

Comment:
```suggestion
     - Adds the delivery guide (Guía de Remisión), which is needed as proof that you are sending
     goods between A and B. It is only when a delivery order is validated that the delivery guide
     can be created.
```
Phrasing this bullet point like this gives it parallel structure to the other bullet points. I also removed an unnecessary "you" from the second sentence. Feel free to check out our [documentation guidelines](https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html) for more information.

Reply:
Wonderful :1st_place_medal: thanks @dade-odoo